server/src/command_table_start.go

```diff
@@ -65,6 +67,11 @@ func commandTableStart(ctx context.Context, s *Session, d *CommandData) {
 		}
 	}
 
+	if now.Sub(t.DatetimeLastJoined) < time.Millisecond*1500 || now.Sub(t.DatetimeLastLeft) < time.Millisecond*1500 {
```

Comment:
try this:

```go
recentTimeThreshold := 1.5 * time.Second
playersRecentlyChanged := now.Sub(t.DatetimeLastJoined) < recentTimeThreshold  || now.Sub(t.DatetimeLastLeft) < recentTimeThreshold 
if playersRecentlyChanged  {
```

Reply:
I doubt you can multiply a duration by a float

Reply:
if you can't multiply a duration by a float, then do:

```go
recentTimeThreshold := 1.5 * time.Millisecond
playersRecentlyChanged := now.Sub(t.DatetimeLastJoined) < recentTimeThreshold || now.Sub(t.DatetimeLastLeft) < recentTimeThreshold 
if playersRecentlyChanged {
```